kernel/cClosure.ml

```diff
@@ -385,8 +386,8 @@ let mk_red f = {norm=Red;term=f}
 
 (* Could issue a warning if no is still Red, pointing out that we loose
    sharing. *)
-let update v1 no t =
-  if !share then
+let update share v1 no t =
```

Comment:
Should be a label.

Reply:
I actually wrote the code with a label the first time, but after pondering it a bit, I chose to remove it. The reason is that these functions are internal to the CClosure module, and there is little point in adding a label in that case, except for adding noise.